### PR TITLE
i18n: Add Swedish

### PR DESCRIPTION
**Pull request by Newbytee**
I'm mildly uncertain about is whether it should be sv or sv_SE, but I went with the former as that's how other locales did it.

### CODE REVIEW

**Comment by Newbytee** [comment]
@kattjevfel thoughts?

File: web/packages/extension/build/_locales/sv/messages.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
+{
+    "settings_ruffle_enable": {
+        "message": "Spela upp Flash-innehåll i Ruffle"
+    },
+    "settings_page_ignore_optout": {
+        "message": "Spela upp Flash-innehåll även på sidor som inaktiverar Ruffle"
+    },
+    "settings_ignore_optout": {
+        "message": "Ignorera webbsidekompatibilitetsvarningar"
```
Comment (kattjevfel) [inline web/packages/extension/build/_locales/sv/messages.json:9]: Is there really no better wording than "webbsidekompatibilitetsvarningar"? I can't think of anything that'd be much better but this looks a bit silly.

File: web/packages/extension/build/_locales/sv/messages.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    },
+    "status_init": {
+        "message": "Avläser aktuella fliken…"
```
Comment (kattjevfel) [inline web/packages/extension/build/_locales/sv/messages.json:12]: Should probably be "Avläser aktuell flik..."

File: web/packages/extension/build/_locales/sv/messages.json [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        "message": "Det finns ingen aktiv flik."
+    },
+    "status_tabs_error": {
+        "message": "Ett fel uppstod när den aktuella fliken slogs upp."
+    },
+    "status_message_init": {
+        "message": "Kontrollerar Ruffle-status i aktuella fliken…"
+    },
+    "status_result_running": {
+        "message": "Ruffle är loadad och spelar upp Flash-innehåll i den aktuella filken."
```
Comment (kattjevfel) [inline web/packages/extension/build/_locales/sv/messages.json:24]: loadad --> laddad

File: web/packages/extension/build/_locales/sv/messages.json [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    },
+    "status_result_optout": {
+        "message": "Ruffle är inte laddad för den nuvarande sidan har markerat sig själv som inkompatibel."
```
Comment (kattjevfel) [inline web/packages/extension/build/_locales/sv/messages.json:27]: If you've switched the above to inläst, change this too to make it consistent

File: web/packages/extension/build/_locales/sv/messages.json [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    },
+    "status_result_disabled": {
+        "message": "Ruffle är inte laddad för den stängdes av av användaren."
```
Comment (kattjevfel) [inline web/packages/extension/build/_locales/sv/messages.json:30]: same as above

File: web/packages/extension/build/_locales/sv/messages.json [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+    },
+    "status_result_error": {
+        "message": "Ett fel uppstod letade upp den aktuella flikens instans av Ruffle."
```
Comment (kattjevfel) [inline web/packages/extension/build/_locales/sv/messages.json:33]: Missing some words here? "Ett fel uppstod *när vi* letade upp den aktuella flikens instans av Ruffle."

File: web/packages/extension/build/_locales/sv/messages.json [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+        "message": "Ladda om fliken för att applicera ändringar"
+    },
+    "open_settings_page": {
+        "message": "Öppna inställningarna"
+    },
+    "settings_page": {
+        "message": "Inställningar"
+    },
+    "description": {
+        "message": "Puts Flash back on the web where it belongs."
```
Comment (kattjevfel) [inline web/packages/extension/build/_locales/sv/messages.json:48]: Completely missing translation.
"Sätter tillbaka Flash på webbsidan/hemsidan den tillhör." I'm frankly not sure what word is most popular in 2020, you choose.

**Comment by kattjevfel** [comment]
Never done a review in my life before, even less on githubs interface, but I hope this is of any use.

File: web/packages/extension/build/_locales/sv/messages.json (hunk 1 above)
Comment (Newbytee) [reply web/packages/extension/build/_locales/sv/messages.json:9]: Yeah I was unsure about this one as well. Felt like I was writing German. I can't think of anything better either myself however.

**Comment by Herschel** [comment]
Thank you!

**Pull request merged**